app/experimenter/experiments/migrations/0164_auto_20210401_1758.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 3.1.7 on 2021-04-01 17:58
+
+from django.db import migrations, models
+
+
+def update_changelogs(apps, schema_editor):
+    NimbusChangeLog = apps.get_model("experiments", "NimbusChangeLog")
+    db_alias = schema_editor.connection.alias
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="Review").update(
+        old_status="Draft",
+        old_publish_status="Review",
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="Accepted").update(
+        old_status="Draft",
+        old_publish_status="Waiting",
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(new_status="Review").update(
+        new_status="Draft",
+        new_publish_status="Waiting",
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(new_status="Accepted").update(
+        new_status="Draft",
+        new_publish_status="Waiting",
+    )
```

Comment:
Oh right we should just make sure these are correct by

1) checkout main
2) make refresh
3) make load_dummy_experiments;make load_dummy_experiments;make load_dummy_experiments
4) checkout branch
5) make migrate
6) make up

and look at the dates to make sure the start and end dates are there and working correctly

Reply:
You can also look at the changelog in the admin to make sure they're correct

Reply:
I just tested 100 random experiments and it seemed to be okay 👍